SupportClient/SupportClient/Controllers/FeedbackViewController.swift

```diff
@@ -0,0 +1,219 @@
+//
+//  FeedbackViewController.swift
+//  SupportClient
+//
+//  Created by Cameron Russell on 3/6/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+import DZNEmptyDataSet
+import BSImagePicker
+import Photos
+
+class FeedbackViewController: UIViewController {
+
+    private var attachedErrorFiles = [PHAsset]()
+    private var collectionView: UICollectionView!
+    
+    private let messageLabel = UILabel()
+    private let messageTextField = UITextField()
+    private let typeLabel = UILabel()
+            
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = UIColor.backgroundColor
+        setupNavigationBar()
+        setupCollectionView()
+        setupConstraints()
+    }
+    
+    func setupNavigationBar() {
+        self.navigationController?.navigationBar.barTintColor = UIColor.navigationTintColor
+        let attributes = [
+            NSAttributedString.Key.font: UIFont._21RobotoMedium!,
+            NSAttributedString.Key.foregroundColor: UIColor.titleColor
```

Comment:
Same here for removing `UIColor`

Reply:
Then get the error: Type of expression is ambiguous without more context